### PR TITLE
add navbar to admin v3 advanced settings and venue view

### PR DESCRIPTION
**Pull request by denisdimitrov**
Navigation bar was missing from admin v3's venue view and advanced settings screens. Now it's added and the screens have been slightly modified to have margin at the bottom because the navbar styles cut the content at the bottom.

### CODE REVIEW

File: src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx [hunk 1]
```diff
@@ -77,26 +79,28 @@ export const AdminAdvancedSettings: React.FC = () => {
 
   return (
     <AdminRestricted>
-      <div className="AdminAdvancedSettings">
-        <div className="AdminAdvancedSettings__options">
-          {renderAdminAdvancedTabs}
+      <WithNavigationBar hasBackButton={false} withSchedule={false}>
```
Comment (goran-peoski-work) [inline src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx:82]: `WithNavigationBar` outside, `AdminRestricted` inside, navigation shouldn't be restricted only to admins

File: src/components/organisms/AdminVenueView/AdminVenueView.tsx [hunk 2]
```diff
@@ -106,26 +108,28 @@ export const AdminVenueView: React.FC = () => {
...
-      {selectedTab === AdminVenueTab.timing && (
-        <Timing
-          onClickHome={navigateToHome}
-          onClickBack={navigateToSpaces}
-          onClickNext={navigateToRun}
-          venue={venue}
-        />
-      )}
-      {selectedTab === AdminVenueTab.run && <RunTabView venue={venue} />}
+      <WithNavigationBar hasBackButton={false} withSchedule={false}>
```
Comment (goran-peoski-work) [inline src/components/organisms/AdminVenueView/AdminVenueView.tsx:111]: `WithNavigationBar` outside, `AdminRestricted` inside, navigation shouldn't be restricted only to admins

File: src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx [hunk 3]
```diff
@@ -335,7 +337,11 @@ const AdvancedSettings: React.FC<AdvancedSettingsProps> = ({
 
         {renderRoomVisibility()}
 
-        <Button type="submit" disabled={!dirty || isSubmitting}>
+        <Button
+          className="AdvancedSettings__save-button"
+          type="submit"
+          disabled={!dirty || isSubmitting}
+        >
```
Comment (goran-peoski-work) [inline src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx:344]: Maybe OOS, but if there is a fast few minute ButtonNG replacement possible - FTW

File: src/pages/Admin/EntranceExperience/EntranceExperience.tsx [hunk 4]
```diff
@@ -138,7 +140,9 @@ const EntranceExperience: React.FC<EntranceExperienceProps> = ({
           </S.ItemBody>
         </S.ItemWrapper>
 
-        <Button type="submit">Save</Button>
+        <Button className="EntranceExperience__save-button" type="submit">
+          Save
+        </Button>
```
Comment (goran-peoski-work) [inline src/pages/Admin/EntranceExperience/EntranceExperience.tsx:145]: Maybe OOS, but if there is a fast few minute ButtonNG replacement possible - FTW

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
Most of the comments about having removed `WithNavigationBar` or moved inide `AdminRestricted` which will limit navigation (out of that location and/or logout) to non-admins.

File: src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx (hunk 1 above)
Comment (denisdimitrov) [reply src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx]: Fixed in: ea4802ee1338ca4c0bf535218a1562a1d7c45bf3

File: src/components/organisms/AdminVenueView/AdminVenueView.tsx (hunk 2 above)
Comment (denisdimitrov) [reply src/components/organisms/AdminVenueView/AdminVenueView.tsx]: Fixed in: ea4802ee1338ca4c0bf535218a1562a1d7c45bf3

File: src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx (hunk 3 above)
Comment (denisdimitrov) [reply src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx]: Fixed in: ea4802ee1338ca4c0bf535218a1562a1d7c45bf3

File: src/pages/Admin/EntranceExperience/EntranceExperience.tsx (hunk 4 above)
Comment (denisdimitrov) [reply src/pages/Admin/EntranceExperience/EntranceExperience.tsx]: Fixed in: ea4802ee1338ca4c0bf535218a1562a1d7c45bf3

File: src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx [hunk 5]
```diff
@@ -40,11 +40,9 @@ const VenueWizardCreate: React.FC<VenueWizardCreateProps> = ({
   }
 
   return (
-    <WithNavigationBar>
```
[the remark this answers is not in the document]
Comment (denisdimitrov) [reply src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx:43]: WithNavigationBar is just removed here.

File: src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx [hunk 6]
```diff
@@ -335,9 +338,13 @@ const AdvancedSettings: React.FC<AdvancedSettingsProps> = ({
 
         {renderRoomVisibility()}
 
-        <Button type="submit" disabled={!dirty || isSubmitting}>
+        <ButtonNG
+          className="AdvancedSettings__save-button"
+          type="submit"
+          disabled={!dirty || isSubmitting}
+        >
```
Comment (goran-peoski-work) [inline src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx:345]:
```suggestion
        <ButtonNG
          className="AdvancedSettings__save-button"
          type="submit"
          loading={isSubmitting}
          disabled={!dirty || isSubmitting}
        >
```

**Review by goran-peoski-work — APPROVED** [review]
LGTM.

Footer is there, NavBar is there. What's between them... 🤪 can be worked on in other admin related PRs 😄

**Pull request merged**